tools/create_basis_compressed_glbs.py

```diff
@@ -0,0 +1,253 @@
+import argparse
+import glob
+import itertools
+import json
+import multiprocessing
+import os
+import os.path as osp
+import shlex
+import shutil
+import subprocess
+from typing import Any, Callable, List, Optional, Tuple
+
+import tqdm
+
+TOOL_PATH = osp.realpath(
+    osp.join(
+        osp.dirname(__file__),
+        "..",
+        "src/deps/magnum-plugins/src/MagnumPlugins/TinyGltfImporter/Test",
+    )
+)
+
+IMAGE_CONVERTER = osp.realpath(
+    osp.join(
+        osp.dirname(__file__), "..", "build/utils/imageconverter/magnum-imageconverter"
```

Comment:
In my experience using this script, I have to locally modify here because I don't usually name my build directory "build". It would be nice to offer a friendly error message like "build directory not found" for users like me.